bindings/pyroot_experimental/PyROOT/src/PyROOTModule.cxx

```diff
@@ -40,15 +40,17 @@ PyObject *gRootModule = 0;
 // Methods offered by the interface
 static PyMethodDef gPyROOTMethods[] = {{(char *)"AddDirectoryWritePyz", (PyCFunction)PyROOT::AddDirectoryWritePyz, METH_VARARGS,
                                         (char *)"Allow to use seamlessly from Python the templated TDirectory::WriteObject method"},
+                                       {(char *)"AddCPPInstancePickling", (PyCFunction)PyROOT::AddCPPInstancePickling, METH_VARARGS,
+                                        (char *)"Implement the function __reduce__ for every PyObject"},
```

Comment:
Here in the description I would rather say: "Add a custom pickling mechanism for Cppyy Python proxy objects"